starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -118,6 +118,8 @@ public void run() {
   private static final TimeZone DEFAULT_TIME_ZONE = TimeZone.getTimeZone("America/Los_Angeles");
   private final long timeNanosecondsPerMicrosecond = 1000;
 
+  private final boolean disableBackgroundPlayback = true;
```

Comment:
Can we make a way for the web app to change this value, e.g. though a Cobalt extension, so that this can be re-enabled in experiments without needing a binary app update?

Reply:
From my testing yesterday, this flag (disableBackgroundPlayback) is not the critical part for fixing the bug, the critical part is the flag (android:enabled) changed in the AndroidManifest file. If android:enabled is set to true, when the app is turned into the background, even though we do not run the code to fire the Intent to start the service, it still occasionally show me the audio tile for a short period of time. 

I could not find a way to dynamically change that flag. And someone said it is not possible here. https://stackoverflow.com/questions/43199178/is-it-possible-to-update-android-manifest-programmatically-at-runtime

Reply:
I think that just having the service should not *directly* determine whether or not there is the 'audio tile', because it's a side-effect of having the service.

That 'audio tile' is really an app notification created and managed by the android `MediaSession`, so it's possible that displaying of the notification can be avoided or more rapidly stopped with the right and/or a more direct action on `CobaltMediaSession.mediaSession`.

Reply:
It is not related to CobaltMediaSession. See the mediaPlayback keyword below.
    <service android:name="dev.cobalt.coat.MediaPlaybackService"
      android:enabled="true"
      android:exported="false"
      android:foregroundServiceType="mediaPlayback"/>
I feel once the service object is created by Android OS, it shows the tile automatically.

Reply:
`MediaPlaybackService` is not where the notification comes from. While not having the `MediaPlaybackService` may also remove the tile, it's not the canonical origin of it.

I've never seen the notification when backgrounding the app while no media was playing (specifically while there is no web MediaSession that in turn results in an android MediaSession being instantiated). The notification is a side effect of the android MediaSession.

Reply:
Great, thanks for the input. I revised the code. Let me know if you are good with the using the flag.

If so, we can add a Cobalt extension to change the value.

Reply:
Updated the code, added the h5vcc settings flag to enable it.

Reply:
Thanks this looks great!